src/components/buttons/AddTxDropdown.tsx

```diff
@@ -48,7 +48,10 @@ export default function AddTxDropdown({
                 {
                   date: (latestDate || DateTime.now()).toISODate() as string,
                   description: '',
-                  splits: [Split.create({ fk_account: account }), new Split()],
+                  splits: [
+                    Split.create({ fk_account: account }),
+                    {},
```

Comment:
passing a split object makes it that when we select an account for this split, the `defaultValues` object is also updated. Setting it to a raw object so this doesn't happen.